style.css

```diff
@@ -929,6 +929,11 @@
     margin-bottom: -55px;
   }
 
+  #mail-message {
+    font-size: 16px;
+    color: rgb(66, 82, 110);
+  }
+
```

Comment:
https://github.com/SaboorSohaib/Protfilo/blob/2a7c0316933ea90b9fa6900b214f2cda672190e9/style.css#L1-L6

- [ ] **[Optional:]** Disable linter is not a good practice. Kindly fix it. :+1: